### PR TITLE
python311Packages.wsgi-intercept: disable failing tests

### PR DESCRIPTION
**Pull request by MaxwellDupre**
## Description of changes
With the updated packages in unstable the current version of wsgi-intercept fails.
This update fixes those issues.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### Priorities

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 269161` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.wsgi-intercept</li>
    <li>python310Packages.wsgi-intercept.dist</li>
    <li>python311Packages.wsgi-intercept</li>
    <li>python311Packages.wsgi-intercept.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/wsgi-intercept/default.nix [hunk 1]
```diff
@@ -36,6 +41,8 @@ buildPythonPackage rec {
   ];
 
   disabledTests = [
+    "test_urllib3"
```
Comment (fabaff) [inline pkgs/development/python-modules/wsgi-intercept/default.nix:44]: A comment would be helpful for the future. Some like:

```suggestion
    # Tests require network access
    "test_urllib3"
```

Or if possible the actual issue.

**Review by fabaff — COMMENTED** [review]
Could you please update the commit message to

```bash
python311Packages.wsgi-intercept: disable failing tests
```
? Thanks

File: pkgs/development/python-modules/wsgi-intercept/default.nix (hunk 1 above)
Comment (MaxwellDupre) [reply pkgs/development/python-modules/wsgi-intercept/default.nix:44]: ok, will do.

**Comment by fabaff** [comment]
I was talking about the commit messages. You changed the PR title.

`Update default.nix`-> `python311Packages.wsgi-intercept: disable failing tests`

**Comment by a-n-n-a-l-e-e** [comment]
@ofborg build python311Packages.wsgi-intercept

**Review by fabaff — DISMISSED** [review]
Commit message of [1ca38e4](https://github.com/NixOS/nixpkgs/pull/269161/commits/1ca38e439a88c572d40126b52d6d961b9da670cc) should be 

```bash
python311Packages.wsgi-intercept: disable failing tests
```

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by MaxwellDupre** [comment]
Requested changes done.

**Comment by fabaff** [comment]
Commit message doesn't follow the guidelines.

**Comment by MaxwellDupre** [comment]
Changed commit message as requested.
Ready for review.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
A bunch of other things fail currently on unstable. Please rebase ot a more current version and try working with that. Maybe an update is required.
Also the commit message isn't fixed

```
wsgi-intercept> ============================= test session starts ==============================
wsgi-intercept> platform linux -- Python 3.11.8, pytest-7.4.4, pluggy-1.4.0
wsgi-intercept> rootdir: /build/wsgi_intercept-1.12.1
wsgi-intercept> collected 81 items / 9 deselected / 72 selected
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_http_client.py .....                           [  6%]
wsgi-intercept> wsgi_intercept/tests/test_httplib2.py ........                           [ 18%]
wsgi-intercept> wsgi_intercept/tests/test_interceptor.py ....................            [ 45%]
wsgi-intercept> wsgi_intercept/tests/test_module_interceptor.py ..                       [ 48%]
wsgi-intercept> wsgi_intercept/tests/test_requests.py .....FFF.                          [ 61%]
wsgi-intercept> wsgi_intercept/tests/test_response_headers.py ..                         [ 63%]
wsgi-intercept> wsgi_intercept/tests/test_urllib.py .......                              [ 73%]
wsgi-intercept> wsgi_intercept/tests/test_urllib3.py .....FFF.                           [ 86%]
wsgi-intercept> wsgi_intercept/tests/test_wsgi_compliance.py .........                   [ 98%]
wsgi-intercept> wsgi_intercept/tests/test_wsgiref.py .                                   [100%]
wsgi-intercept>
wsgi-intercept> =================================== FAILURES ===================================
wsgi-intercept> __________________________________ test_https __________________________________
wsgi-intercept>
wsgi-intercept>     def test_https():
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=443) as app:
wsgi-intercept> >           resp = requests.get('https://some_hopefully_nonexistant_domain:443/')
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_requests.py:57:
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/api.py:73: in get
wsgi-intercept>     return request("get", url, params=params, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/api.py:59: in request
wsgi-intercept>     return session.request(method=method, url=url, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/sessions.py:589: in request
wsgi-intercept>     resp = self.send(prep, **send_kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/sessions.py:703: in send
wsgi-intercept>     r = adapter.send(request, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/adapters.py:486: in send
wsgi-intercept>     resp = conn.urlopen(
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:769: in urlopen
wsgi-intercept>     conn = self._get_conn(timeout=pool_timeout)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:295: in _get_conn
wsgi-intercept>     return conn or self._new_conn()
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:1070: in _new_conn
wsgi-intercept>     return self.ConnectionCls(
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept>
wsgi-intercept> self = <wsgi_intercept._urllib3.make_urllib3_override.<locals>.HTTPS_WSGIInterceptor object at 0x7ffff57884d0>
wsgi-intercept> args = ()
wsgi-intercept> kwargs = {'assert_fingerprint': None, 'assert_hostname': None, 'blocksize': 16384, 'ca_cert_dir': None, ...}
wsgi-intercept>
wsgi-intercept>     def __init__(self, *args, **kwargs):
wsgi-intercept>         if 'strict' in kwargs and sys.version_info > (3, 0):
wsgi-intercept>             kwargs.pop('strict')
wsgi-intercept>         kwargs.pop('socket_options', None)
wsgi-intercept>         kwargs.pop('key_password', None)
wsgi-intercept>         kwargs.pop('server_hostname', None)
wsgi-intercept> >       WSGI_HTTPSConnection.__init__(self, *args, **kwargs)
wsgi-intercept> E       TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_reqs'
wsgi-intercept>
wsgi-intercept> wsgi_intercept/_urllib3.py:33: TypeError
wsgi-intercept> __________________ test_https_no_ssl_verification_intercepted __________________
wsgi-intercept>
wsgi-intercept>     def test_https_no_ssl_verification_intercepted():
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=443) as app:
wsgi-intercept> >           resp = requests.get('https://some_hopefully_nonexistant_domain:443/',
wsgi-intercept>                                 verify=False)
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_requests.py:64:
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/api.py:73: in get
wsgi-intercept>     return request("get", url, params=params, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/api.py:59: in request
wsgi-intercept>     return session.request(method=method, url=url, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/sessions.py:589: in request
wsgi-intercept>     resp = self.send(prep, **send_kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/sessions.py:703: in send
wsgi-intercept>     r = adapter.send(request, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/adapters.py:486: in send
wsgi-intercept>     resp = conn.urlopen(
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:769: in urlopen
wsgi-intercept>     conn = self._get_conn(timeout=pool_timeout)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:295: in _get_conn
wsgi-intercept>     return conn or self._new_conn()
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:1070: in _new_conn
wsgi-intercept>     return self.ConnectionCls(
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept>
wsgi-intercept> self = <wsgi_intercept._urllib3.make_urllib3_override.<locals>.HTTPS_WSGIInterceptor object at 0x7ffff564e350>
wsgi-intercept> args = ()
wsgi-intercept> kwargs = {'assert_fingerprint': None, 'assert_hostname': None, 'blocksize': 16384, 'ca_cert_dir': None, ...}
wsgi-intercept>
wsgi-intercept>     def __init__(self, *args, **kwargs):
wsgi-intercept>         if 'strict' in kwargs and sys.version_info > (3, 0):
wsgi-intercept>             kwargs.pop('strict')
wsgi-intercept>         kwargs.pop('socket_options', None)
wsgi-intercept>         kwargs.pop('key_password', None)
wsgi-intercept>         kwargs.pop('server_hostname', None)
wsgi-intercept> >       WSGI_HTTPSConnection.__init__(self, *args, **kwargs)
wsgi-intercept> E       TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_reqs'
wsgi-intercept>
wsgi-intercept> wsgi_intercept/_urllib3.py:33: TypeError
wsgi-intercept> ___________________________ test_https_default_port ____________________________
wsgi-intercept>
wsgi-intercept>     def test_https_default_port():
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=443) as app:
wsgi-intercept> >           resp = requests.get('https://some_hopefully_nonexistant_domain/')
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_requests.py:80:
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/api.py:73: in get
wsgi-intercept>     return request("get", url, params=params, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/api.py:59: in request
wsgi-intercept>     return session.request(method=method, url=url, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/sessions.py:589: in request
wsgi-intercept>     resp = self.send(prep, **send_kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/sessions.py:703: in send
wsgi-intercept>     r = adapter.send(request, **kwargs)
wsgi-intercept> /nix/store/va7cjgv91lkcja8jz846rd646d14sjbw-python3.11-requests-2.31.0/lib/python3.11/site-packages/requests/adapters.py:486: in send
wsgi-intercept>     resp = conn.urlopen(
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:769: in urlopen
wsgi-intercept>     conn = self._get_conn(timeout=pool_timeout)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:295: in _get_conn
wsgi-intercept>     return conn or self._new_conn()
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:1070: in _new_conn
wsgi-intercept>     return self.ConnectionCls(
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept>
wsgi-intercept> self = <wsgi_intercept._urllib3.make_urllib3_override.<locals>.HTTPS_WSGIInterceptor object at 0x7ffff54b3190>
wsgi-intercept> args = ()
wsgi-intercept> kwargs = {'assert_fingerprint': None, 'assert_hostname': None, 'blocksize': 16384, 'ca_cert_dir': None, ...}
wsgi-intercept>
wsgi-intercept>     def __init__(self, *args, **kwargs):
wsgi-intercept>         if 'strict' in kwargs and sys.version_info > (3, 0):
wsgi-intercept>             kwargs.pop('strict')
wsgi-intercept>         kwargs.pop('socket_options', None)
wsgi-intercept>         kwargs.pop('key_password', None)
wsgi-intercept>         kwargs.pop('server_hostname', None)
wsgi-intercept> >       WSGI_HTTPSConnection.__init__(self, *args, **kwargs)
wsgi-intercept> E       TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_reqs'
wsgi-intercept>
wsgi-intercept> wsgi_intercept/_urllib3.py:33: TypeError
wsgi-intercept> __________________________________ test_https __________________________________
wsgi-intercept>
wsgi-intercept>     def test_https():
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=443) as app:
wsgi-intercept> >           resp = http.request(
wsgi-intercept>                 'GET', 'https://some_hopefully_nonexistant_domain:443/')
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_urllib3.py:59:
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/_request_methods.py:110: in request
wsgi-intercept>     return self.request_encode_url(
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/_request_methods.py:143: in request_encode_url
wsgi-intercept>     return self.urlopen(method, url, **extra_kw)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/poolmanager.py:444: in urlopen
wsgi-intercept>     response = conn.urlopen(method, u.request_uri, **kw)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:769: in urlopen
wsgi-intercept>     conn = self._get_conn(timeout=pool_timeout)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:295: in _get_conn
wsgi-intercept>     return conn or self._new_conn()
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:1070: in _new_conn
wsgi-intercept>     return self.ConnectionCls(
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept>
wsgi-intercept> self = <wsgi_intercept._urllib3.make_urllib3_override.<locals>.HTTPS_WSGIInterceptor object at 0x7ffff5d8af50>
wsgi-intercept> args = ()
wsgi-intercept> kwargs = {'assert_fingerprint': None, 'assert_hostname': None, 'blocksize': 16384, 'ca_cert_dir': None, ...}
wsgi-intercept>
wsgi-intercept>     def __init__(self, *args, **kwargs):
wsgi-intercept>         if 'strict' in kwargs and sys.version_info > (3, 0):
wsgi-intercept>             kwargs.pop('strict')
wsgi-intercept>         kwargs.pop('socket_options', None)
wsgi-intercept>         kwargs.pop('key_password', None)
wsgi-intercept>         kwargs.pop('server_hostname', None)
wsgi-intercept> >       WSGI_HTTPSConnection.__init__(self, *args, **kwargs)
wsgi-intercept> E       TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_reqs'
wsgi-intercept>
wsgi-intercept> wsgi_intercept/_urllib3.py:33: TypeError
wsgi-intercept> ___________________________ test_https_default_port ____________________________
wsgi-intercept>
wsgi-intercept>     def test_https_default_port():
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=443) as app:
wsgi-intercept> >           resp = http.request(
wsgi-intercept>                 'GET', 'https://some_hopefully_nonexistant_domain/')
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_urllib3.py:67:
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/_request_methods.py:110: in request
wsgi-intercept>     return self.request_encode_url(
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/_request_methods.py:143: in request_encode_url
wsgi-intercept>     return self.urlopen(method, url, **extra_kw)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/poolmanager.py:444: in urlopen
wsgi-intercept>     response = conn.urlopen(method, u.request_uri, **kw)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:769: in urlopen
wsgi-intercept>     conn = self._get_conn(timeout=pool_timeout)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:295: in _get_conn
wsgi-intercept>     return conn or self._new_conn()
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:1070: in _new_conn
wsgi-intercept>     return self.ConnectionCls(
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept>
wsgi-intercept> self = <wsgi_intercept._urllib3.make_urllib3_override.<locals>.HTTPS_WSGIInterceptor object at 0x7ffff57a66d0>
wsgi-intercept> args = ()
wsgi-intercept> kwargs = {'assert_fingerprint': None, 'assert_hostname': None, 'blocksize': 16384, 'ca_cert_dir': None, ...}
wsgi-intercept>
wsgi-intercept>     def __init__(self, *args, **kwargs):
wsgi-intercept>         if 'strict' in kwargs and sys.version_info > (3, 0):
wsgi-intercept>             kwargs.pop('strict')
wsgi-intercept>         kwargs.pop('socket_options', None)
wsgi-intercept>         kwargs.pop('key_password', None)
wsgi-intercept>         kwargs.pop('server_hostname', None)
wsgi-intercept> >       WSGI_HTTPSConnection.__init__(self, *args, **kwargs)
wsgi-intercept> E       TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_reqs'
wsgi-intercept>
wsgi-intercept> wsgi_intercept/_urllib3.py:33: TypeError
wsgi-intercept> _____________________________ test_socket_options ______________________________
wsgi-intercept>
wsgi-intercept>     def test_socket_options():
wsgi-intercept>         http = urllib3.PoolManager(socket_options=[])
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=80):
wsgi-intercept>             http.request('GET', 'http://some_hopefully_nonexistant_domain/')
wsgi-intercept>         with InstalledApp(wsgi_app.simple_app, host=HOST, port=443):
wsgi-intercept> >           http.request('GET', 'https://some_hopefully_nonexistant_domain/')
wsgi-intercept>
wsgi-intercept> wsgi_intercept/tests/test_urllib3.py:80:
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/_request_methods.py:110: in request
wsgi-intercept>     return self.request_encode_url(
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/_request_methods.py:143: in request_encode_url
wsgi-intercept>     return self.urlopen(method, url, **extra_kw)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/poolmanager.py:444: in urlopen
wsgi-intercept>     response = conn.urlopen(method, u.request_uri, **kw)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:769: in urlopen
wsgi-intercept>     conn = self._get_conn(timeout=pool_timeout)
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:295: in _get_conn
wsgi-intercept>     return conn or self._new_conn()
wsgi-intercept> /nix/store/qb2kp8814k84rxhs0byhl6cgyfz1gljp-python3.11-urllib3-2.1.0/lib/python3.11/site-packages/urllib3/connectionpool.py:1070: in _new_conn
wsgi-intercept>     return self.ConnectionCls(
wsgi-intercept> _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
wsgi-intercept>
wsgi-intercept> self = <wsgi_intercept._urllib3.make_urllib3_override.<locals>.HTTPS_WSGIInterceptor object at 0x7ffff5465c10>
wsgi-intercept> args = ()
wsgi-intercept> kwargs = {'assert_fingerprint': None, 'assert_hostname': None, 'blocksize': 16384, 'ca_cert_dir': None, ...}
wsgi-intercept>
wsgi-intercept>     def __init__(self, *args, **kwargs):
wsgi-intercept>         if 'strict' in kwargs and sys.version_info > (3, 0):
wsgi-intercept>             kwargs.pop('strict')
wsgi-intercept>         kwargs.pop('socket_options', None)
wsgi-intercept>         kwargs.pop('key_password', None)
wsgi-intercept>         kwargs.pop('server_hostname', None)
wsgi-intercept> >       WSGI_HTTPSConnection.__init__(self, *args, **kwargs)
wsgi-intercept> E       TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'cert_reqs'
wsgi-intercept>
wsgi-intercept> wsgi_intercept/_urllib3.py:33: TypeError
wsgi-intercept> =========================== short test summary info ============================
wsgi-intercept> FAILED wsgi_intercept/tests/test_requests.py::test_https - TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'c...
wsgi-intercept> FAILED wsgi_intercept/tests/test_requests.py::test_https_no_ssl_verification_intercepted - TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'c...
wsgi-intercept> FAILED wsgi_intercept/tests/test_requests.py::test_https_default_port - TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'c...
wsgi-intercept> FAILED wsgi_intercept/tests/test_urllib3.py::test_https - TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'c...
wsgi-intercept> FAILED wsgi_intercept/tests/test_urllib3.py::test_https_default_port - TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'c...
wsgi-intercept> FAILED wsgi_intercept/tests/test_urllib3.py::test_socket_options - TypeError: HTTPSConnection.__init__() got an unexpected keyword argument 'c...
wsgi-intercept> ================== 6 failed, 66 passed, 9 deselected in 0.42s ==================
```

**Comment by MaxwellDupre** [comment]
I ma having problems re basing.

**Comment by MaxwellDupre** [comment]
I am thinking of closing this PR in favor of [python311Packages.wsgi-intercept: 1.12.1->1.13.0](https://github.com/NixOS/nixpkgs/pull/293530)

**Comment by fabaff** [comment]
#293530 was merged.

**Pull request closed without merging**